app/models/description_change_validation_request.rb

```diff
@@ -59,6 +61,10 @@ def audit_cancel!
     )
   end
 
+  def response_due
+    RESPONSE_TIME_IN_DAYS.business_days.after(created_at).to_date
```

Comment:
I tried overwriting the constant rather than the whole method, but it seems that a method defined in a concern will pick up the constant from that concern, not from the class it's included in 🤔